Tests/AlicerceTests/Stores/StoreTestCase.swift

```diff
@@ -76,18 +86,58 @@ class StoreTestCase: XCTestCase {
 
     // MARK: Failure
 
+    //     Network Stack: Error
+    // Persistence Stack: Error
```

Comment:
this is misleading, since the `noObjectForKey` is an error which symbolizes normal functioning of the persistence stack ( it's a simple cache miss 😛 )

same for some of the other cases below